### DISCUSSION TITLE
No permissions to access sessions

### DISCUSSION THREAD

**Discussion by thedinz** [Taxel/PlexTraktSync#833] [Q&A] [answered]
I hope this belongs here and not under issues.

I am on unraid and i think i finally got the "watch" command working, the container no longer stops, its stays running but when i open the logs all i get is this

No permission to access sessions, disabling username filter
g for events!
arting AlertListener: wss://192-168-1-215.7b58565fed6c4d46a5c7561d40f22fae.plex.direct:32400/:/websockets/notifications?X-Plex-Token=<hidden>

In case that benign i marked something watched on trakt and never saw the logs update even after waiting some time, so i do not think its a benign error.

Thanks in advanced.

**Comment by glensc** [comment, marked as answer]
"No permissions to access sessions" warniing is happening when you accessing server you don't own. so this might ve expected.

watch command syncs only changes initiated from plex. so changes initiated from trakt do not trigger any action.
